lab_1/build-client.sh

```diff
@@ -0,0 +1,16 @@
+CLIENT_BUILD_DIR=dist
+CLIENT_BUILD_FILE=dist/client-app.zip
+
+if [ -e $CLIENT_BUILD_FILE ]; then
+	rm "$CLIENT_BUILD_FILE"
+	echo "$CLIENT_BUILD_FILE was removed"
+fi
+
+npm install
```

Comment:
just to mention: it's better to have "npm ci" instead of "npm install" in case if we expect the script to be used for CI/CD pipeline